avaframe/com1DFA/com1DFA.py

```diff
@@ -1387,15 +1446,15 @@ def computeEulerTimeStep(cfg, particles, fields, dt, dem, Tcpu, frictType):
         force['forceSPHZ'] = np.zeros(np.shape(force['forceZ']))
     else:
         log.debug('Compute Force SPH C')
-        particles, force = DFAfunC.computeForceSPHC(cfg, particles, force, dem, gradient=0)
+        particles, force = DFAfunC.computeForceSPHC(cfg, particles, force, dem, cfg.getint('sphOption'), gradient=0)
```

Comment:
why? we already pass the cfg as input?

Reply:
because for the ini step we want to use the sphOptionIni and not the sphOption - so we would have to pass another flag instead and then an if inside the function..